plan/config.go

```diff
@@ -59,17 +61,31 @@ func ReadConfigFromEnviron() (*Config, error) {
 
 	var axes Axes
 	var behaviors Behaviors
+	filterMap := make(map[string][]Filter)
 	for _, e := range os.Environ() {
-		if strings.HasPrefix(e, axisKeyPrefix) {
+		switch {
+		case strings.HasPrefix(e, axisKeyPrefix):
 			axis := parseAxis(strings.TrimPrefix(e, axisKeyPrefix))
 			axes = append(axes, axis)
-		} else if strings.HasPrefix(e, behaviorKeyPrefix) {
+			break
+		case strings.HasPrefix(e, skipKeyPrefix):
+			key, filters, err := parseSkipBehavior(strings.TrimPrefix(e, skipKeyPrefix))
+			if err != nil {
+				return nil, err
```

Comment:
More information in the error message please.

    return nil, fmt.Errorf("failed to parse filters from %q: %v", e, err)